src/controllers/trashbin.ts

```diff
@@ -1,11 +1,13 @@
 import { generateUniqueTrashbinIdentifier } from '../service/trashbin';
 import { Project } from '../models/project';
 import { Trashbin } from '../models/trashbin';
+import { History } from '../models/history';
+import { Types } from 'mongoose';
```

Comment:
if you check for example line 151 you can see, that they were already referencing mongoose.Types. So I would propose to either remove this import and do line 411 with "mongoose.Types" or change the other references to just "Types" and removing the import in line 7.
Whichever you prefer